src/isomorphic/modern/class/__tests__/ReactClassEquivalence-test.js

```diff
@@ -32,7 +32,7 @@ describe('ReactClassEquivalence', function() {
 function runJest(testFile) {
   var cwd = process.cwd();
   var jestBin = path.resolve('node_modules', '.bin', 'jest');
-  var setupFile = path.resolve(__dirname, 'setupSpecEquivalenceReporter.js');
+  var setupFile = path.resolve(__dirname, 'setupSpecEquivalenceReporter.js.setup');
```

Comment:
Jest now warns when it runs files that do not have any tests. We shouldn't even run this file through jest but because it is in the __tests__ folder and ends with .js, it is automatically being ran.

Adding .js.setup extension is a quick fix to make it work.